src/pages/Landing.tsx

```diff
@@ -34,7 +34,10 @@ const Tab1: React.FC = () => {
                         <IonTitle size="large">Landing</IonTitle>
                     </IonToolbar>
                 </IonHeader>
-                <ExploreContainer name="Landing page, Ratik says hello!" />
+                <div className="VideoPlaceholder">Video Placeholder</div>
```

Comment:
I think we're trying not to use `div, h2, p` and pure HTML tags in favour for Ionic component since they're more optimized. Can you switch to those instead @Special-Sharpie?

Reply:
yeah ill look into it